### PR TITLE
simple timer #6

### PR DESCRIPTION
**Pull request by ArmenSan**
flexible code for timer which displays how much time is left

### CODE REVIEW

**Comment by fuzzytern** [comment]
@ArmenSan Issue #2 has already been merged. Your team needs to rebase their work on the latest state of upstream/master and work from there. If your group have improvements, they have to work from there, and if these improvements are not directly linked to an issue from the [issue board](https://waffle.io/iteratehackerspace/horloge), they need to create one there before sending a pull request.

Also, @iteratehackerspace/itimer please make sure all members of your team accept they group invite (Only @Mikazakaryan did so far)

File: index.js [hunk 1]
```diff
@@ -5,20 +5,30 @@
...
+  } else if (i % 60 === 0) {
+    if (i / 60 <= 5 && i / 60 > 0) {
+      console.log(`${i / 60} min`);
+    }
+  } else if (i === 90) {
+    console.log(`${(i - (i % 60)) / 60} min ${i % 60} sec`);
+  } else if (i === 30 || i === 20 || i <= 10) {
+    console.log(`${i} sec`);
   }
+  setTimeout(timer, 1000, i - 1);
```
Comment (hayko1995) [inline index.js]: not good idea use setTimeout, use setInterval

Comment (ArmenSan) [inline index.js]: why it's not good idea?

Comment (fxfactorial) [inline index.js]: https://zetafleet.com/blog/2010/04/why-i-consider-setinterval-to-be-harmful.html

Comment (ArmenSan) [inline index.js]: thank you !

**Pull request closed without merging**